rust/opendp-ffi/src/data.rs

```diff
@@ -7,12 +7,12 @@ use opendp::data::Column;
 
 use crate::core::{FfiObject, FfiOwnership, FfiResult, FfiSlice};
 use crate::util;
-use crate::util::{Type, TypeArgs, TypeContents};
+use crate::util::{Type, TypeArgs, TypeContents, c_bool};
 use std::fmt::Debug;
 use opendp::error::Fallible;
 
 #[no_mangle]
-pub extern "C" fn opendp_data__object_new(type_args: *const c_char, raw: *const FfiSlice) -> FfiResult<*mut FfiObject> {
+pub extern "C" fn opendp_data__slice_as_object(type_args: *const c_char, raw: *const FfiSlice) -> FfiResult<*mut FfiObject> {
```

Comment:
I like this renaming, it's more explicit about slices.

Reply:
I kept on getting them backwards. I figured I wouldn't be the only one.